modules/nextflow/src/main/groovy/nextflow/k8s/K8sTaskHandler.groovy

```diff
@@ -216,6 +218,14 @@ class K8sTaskHandler extends TaskHandler implements FusionAwareTask {
             .withLabels(getLabels(task))
             .withAnnotations(getAnnotations())
             .withPodOptions(getPodOptions())
+            .withHostMount("/opt/latch-env", "/opt/latch-env")
```

Comment:
wonder if it makes more sense to mount this in the dispatcher instead? Mostly because this will enable us to change/remove dynamically without having to support backward compatibility with existing workflows

Reply:
probably, itll still be backwards compatible if we move it to the dispatcher bc we can just clear the mounts and remake them w/ whatever new stuff so im not too concerned